www/src/pages/ja/introduction.md

```diff
@@ -0,0 +1,74 @@
+
+お気づきかもしれませんが、構成要素の数が多いのですが、意図的にそうしており、必要に応じて構成要素を入れ替えることができます。このスタックはモジュール式です。:）
+
+## So... what is create-t3-app? A template?
+
+Kind of? `create-t3-app` is a CLI built by seasoned T3 Stack devs to streamline the setup of a modular T3 Stack app. This means each piece is optional, and the "template" is generated based on your specific needs.
+
+## さて...create-t3-app とは何でしょう？テンプレート？
+
+一種そのようなものです。`create-t3-app`は、モジュール式の T3 Stack アプリのセットアップを効率化するために、経験豊富な T3 Stack 開発者によって作られた CLI です。つまり、各パーツがオプションであり、「テンプレート」があなたの特定のニーズに基づいて生成されることを意味します。
+
+After countless projects and many years on this tech, we have lots of opinions and insights. We've done our best to encode them into this CLI.
+
+数え切れないほどのプロジェクトと長年この技術に携わってきた私たちには、たくさんの意見や洞察があります。私たちは、それらをこの CLI に反映させるために最善を尽くしました。
+
+This is **NOT** an all-inclusive template. We **expect** you to bring your own libraries that solve the needs of **YOUR** application. While we don't want to prescribe solutions to more specific problems like state management and deployment, we [do have some recommendations listed here](/en/other-recs).
+
+ただこれは、すべてを網羅したテンプレートでは**ありません**。私たちは、あなたが **あなたの**アプリケーションのニーズを解決する独自のライブラリを持ってくることを**期待しています**。状態管理やデプロイメントなど、より具体的な問題に対する解決策を規定するつもりはありませんが、私たちは[ここにいくつかの推奨事項を挙げています](/en/other-recs).
+
+## T3 Axioms
+
+We'll be frank - this is an _opinionated project_. We share a handful of core beliefs around building and we treat them as the basis for our decisions.
+
+## T3 原則
+
+率直に言いますが、T3 Stack は「_主張の強い_」プロジェクトです。私たちは、開発に関する一握りの核となる信念を共有し、それを私たちの判断基準としています。
+
+### Solve Problems
+
+It's easy to fall into the trap of "adding everything" - we explicitly don't want to do that. Everything added to `create-t3-app` should solve a specific problem that exists within the core technologies included. This means we won't add things like state libraries (`zustand`, `redux`) but we will add things like NextAuth.js and integrate Prisma and tRPC for you.
+
+### 問題を解決する
+
+「何でもかんでも追加してしまう」という罠に容易に陥りがちですが、わたしたちは明示的にそれを避けます。`create-t3-app` に追加するものはすべて、その領域における固有の問題を解決している必要があります。したがって状態ライブラリ(Zustand、Redux 等)は含めませんが、Auth.js などは追加し、Prisma や tRPC などを統合します。
+
+### Bleed Responsibly
+
+We love our bleeding edge tech. The amount of speed and, honestly, fun that comes out of new shit is really cool. We think it’s important to bleed responsibly, using riskier tech in the less risky parts. This means we wouldn’t ⛔️ bet on risky new database tech (SQL is great!). But we happily ✅ bet on tRPC since it’s just functions that are trivial to move off.
+
+### 責任を負って血を流す
+
+私たちは、最先端の技術を愛しています。正直なところ、新しい技術から生まれる開発速度と楽しさは本当に素晴しいものです。ただし、責任を持って血を流すことが重要であり、リスクが低い領域では高リスク技術を採用します。なので冒険的な新しいデータベース技術は採用しませんが(SQL は偉大なり!)。しかし、tRPC は単なる機能であり、問題があれば容易に交換できるので、私たちは喜んで賭けます。
```

Comment:
```suggestion
私たちは、最先端の技術を愛しています。正直なところ、新しい技術から生まれる開発速度と楽しさは本当に素晴しいものです。ただし、責任を持って血を流すことが重要であり、リスクが低い領域では高リスク技術を採用します。なので冒険的な新しいデータベース技術は採用しませんが(SQL は偉大なり!)。しかし、tRPC は単なる関数であり、問題があれば容易に交換できるので、私たちは喜んで賭けます。
```
「最先端」には「血」という比喩が含めていないのに「責任を持って血を流す」の意味がわかりやすく伝えられていますか。

Reply:
Thank you for your comment.

I translated the word "function" as "機能" in Japanese. 機能 means "the purpose of something or someone" (https://dictionary.cambridge.org/ja/dictionary/learner-english/function).

Are you saying that in this context, tRPC being a function means it is a function in a programming language?

Reply:
> 「最先端」には「血」という比喩が含めていないのに「責任を持って血を流す」の意味がわかりやすく伝えられていますか。

Translating "bleed responsibly" was very challenging in this context. Such expressions don't exist in Japanese, and I think the same is true in English.

However, I can grasp the nuance. In Japanese, "shedding blood" also means making sacrifices. I tried to translate this part carefully so that this nuance could be understood.

Reply:
> Thank you for your comment.
> 
> I translated the word "function" as "機能" in Japanese. 機能 means "faculty, mechanism" . And "関数" in Japanese refers to the concept of a function in programming languages or a mathematical function.
> 
> Are you saying that in this context, tRPC being a "関数" means it is a function in a programming language?

Yes, the intended meaning of "tRPC is just functions" is to say that adopting tRPC is low risk, because procedures are just functions (`({ ctx, input }) => res`) so it's easy to switch to Express, Fastify, etc. in the future.